### ISSUE TITLE
Model Selection on BMNIST

### ISSUE THREAD

**Issue by PaulLyonel** [ccr-cheng/statistical-flow-matching#1]
Hey thanks for the cool work! For the sake of comparison I have wondered how you did the model selection for the binary mnist. you said you trained the model for 100k iterations, and then evaluated the fid. did you evaluate the final model (after 100k its), the model with the lowest validation loss or did you check the fid every few steps and reported the best one?

**Comment by ccr-cheng** [comment]
Hi Paul,

Thanks for your interest in our work! For the binarized MNIST dataset, we directly use the last checkpoint at 100k iterations, as we noted that unconditional generative models were generally harder to overfit. We didn't evaluate the FID for the validation steps (mostly because it is somewhat time-consuming), nor did we pick the checkpoint with the best FID scores.

Nonetheless, we noted that, for the conditional generation task on promoter design, we did evaluate the SP-MSE regularly on the valid set to select the best model for test set evaluation. We also observed that such conditional generative models could more easily overfit. Please take a look at Appendix C.6 in our paper for details.

Best,
Chaoran

**Comment by PaulLyonel** [comment]
Many thanks for the clarification!
